src/components/UI/Progress/CircularProgress.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="relative">
+    <svg :height="radius * 2" :width="radius * 2">
+      <!-- background circle -->
+      <circle
+        stroke="#C4C5C5"
+        :stroke-dasharray="circumference + ' ' + circumference"
+        :style="{ strokeDashoffset: 0 }"
+        :stroke-width="stroke"
+        fill="transparent"
+        :r="normalizedRadius"
+        :cx="radius"
+        :cy="radius"
+      />
+      <!-- the actual progress circle -->
+      <circle
+        stroke="#2B9D05"
+        :stroke-dasharray="circumference + ' ' + circumference"
+        :style="{ strokeDashoffset: strokeDashoffset }"
+        :stroke-width="stroke"
+        fill="transparent"
+        :r="normalizedRadius"
+        :cx="radius"
+        :cy="radius"
+      />
+    </svg>
+    <!-- progress indicator in the center of the circle  -->
+    <div
+      v-if="toShowNumberIndicator"
```

Comment:
not sure why we need v-if in the first place - what is the condition when accuracy won't be shown?
```suggestion
      v-if="isAccuracyShown"
```